### ISSUE TITLE
`sameSite` isn't proxied to cookie store when using adaptive store

### ISSUE THREAD

**Issue by makepanic** [simplabs/ember-simple-auth#2230]
Hi,

I was trying to configure the `sameSite` property which was introduced in https://github.com/simplabs/ember-simple-auth/pull/1848 .

It seems like the `sameSite` property isn't proxied to the cookie store when it's created:

https://github.com/simplabs/ember-simple-auth/blob/master/packages/ember-simple-auth/addon/session-stores/adaptive.js#L124

If I simply add `sameSite` to the properties list of `getProperties`, it works as expected. 

Would it be ok if I create a PR to add the property there or do you also want an additional test (it might be a bit complicated because in adaptive store test, the cookie store is created [outside the `_createStore` method](https://github.com/simplabs/ember-simple-auth/blob/master/packages/ember-simple-auth/tests/helpers/create-adaptive-store.js#L12-L22))?

**Comment by marcoow** [comment]
Yes, a PR for that would be cool but I think that'd need a number of things:

* adding the new property to the adaptive store and documenting it
* passing it to the cookie store
* adding a test for that behavior (I know the adaptive store tests are a mess – if you need help with that, let me know)

**Comment by marcoow** [comment]
This was fixed in #2312
